docs/docs/RTB/Analisi dei requisiti.md

```diff
@@ -0,0 +1,77 @@
+# Documento dell'analisi dei requisiti
+
+<details>
+  <summary>Changelog</summary>
+
+| Data | Versione | Descrizione | Autore | Data Verifica | Verificatore |
+|------|----------|-------------|---------|------------------|-------------|
+| 13/12/2024 | 0.1.0 | Prima stesura del documento | Pistori Gaia | ??/12/2024 | ?revisore |
+
+</details>
+
+**1. Introduzione**
+
+    1.1 Scopo del Documento
+
+    1.2. Scopo del Prodotto
+
+**2. Analisi dei requisiti**
+
+	2.1 Requisiti funzionali 
+
+		2.1.1 Requisiti funzionali obbligatori
+		2.1.2 Requisiti funzionali opzionali
+
+	2.2 Requisiti non funzionali 
+		2.2.1 Requisiti non funzionali obbligatori
+		2.2.2 Requisiti non funzionali opzionali
+
+	2.3 Requisiti tecnici 
+		2.3.1 Requisiti tecnici obbligatori
+		2.3.2 Requisiti tecnici opzionali
```

Comment:
Non è necessaria la creazione di un TOC a mano. Diventa difficile da gestire al crescere del documento. Docusaurus implementa già una TOC nella sidebar "laterale", per averne una anche all'inizio del documento è possibile inserire:

```suggestion
import TOCInline from '@theme/TOCInline';
<TOCInline toc={toc} />
```

Eventualmente modificando l'estensione del file in .mdx